lib/matplotlib/cbook.py

```diff
@@ -1408,15 +1408,15 @@ def finddir(o, match, case=False):
 
 def reverse_dict(d):
```

Comment:
I'd rather deprecate this function... I think everyone can understand what `{v: k for k, v in d.items()}` does and this way you don't have to go and check whether the function does something funny with duplicates.